mapie/tests/test_regression.py

```diff
@@ -279,20 +284,16 @@ def test_prediction_between_low_up(strategy: str) -> None:
 
 
 @pytest.mark.parametrize("method", ["plus", "minmax"])
-@pytest.mark.parametrize("cv", [-1, 2, 3, 5])
 @pytest.mark.parametrize("agg_function", ["mean", "median"])
-@pytest.mark.parametrize("alpha", [0.05, 0.1, 0.2])
-def test_prediction_agg_function(
-    method: str, cv: Union[LeaveOneOut, KFold], agg_function: str, alpha: int
-) -> None:
+def test_prediction_agg_function(method: str, agg_function: str) -> None:
```

Comment:
Why simplifying this test ?

Reply:
Because it was one of the most unnecessarily time-consuming.